docs/source/documentation.rst

```diff
@@ -326,7 +326,7 @@ Morphometrics of a single image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Before computing the morphometrics of an image, make sure it has been segmented using AxonDeepSeg ::
```

Comment:
I think the -a argument should be listed as optional argument above.

Reply:
Yes, I agree. I'll fix this.

Reply:
Addressed these changes in the latest commit of this PR.